### PR TITLE
[binary-search/ru] Binary Search Russian translation (Upd)

### PR DESCRIPTION
**Pull request by JustBlah**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.
- [x] Mark the EN resources with a "(EN)", or use russian links.

### CODE REVIEW

File: ru-ru/binary-search-ru.html.markdown [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+category: Algorithms & Data Structures
+name: Binary Search
+contributors:
+    - ["Abhishek Jaisingh", "http://github.com/abhishekjiitr"]
+    - ["Evan K.", "https://github.com/justblah"]
```
Comment (vendethiel) [inline ru-ru/binary-search-ru.html.markdown]: Why?

Comment (JustBlah) [inline ru-ru/binary-search-ru.html.markdown]: @vendethiel contributed with 'Further reading' part at the end.

Comment (vendethiel) [inline ru-ru/binary-search-ru.html.markdown]: That's just another part of the translation

Comment (JustBlah) [inline ru-ru/binary-search-ru.html.markdown]: @vendethiel sounds reasonable to me, will be fixed in a minute.

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**